requirements/docs

```diff
@@ -1,4 +1,5 @@
-docutils
+# TODO(vytas): Unpin when Sphinx is upgraded.
+docutils < 0.18.0
```

Comment:
do we still need this?

Reply:
I've just added this, cause some issue was triggered with an older version of Sphinx that we're using.